src/services/user/account.ts

```
@@ -0,0 +1,66 @@
+  token,
+}: PasswordResetProps): Promise<void | HttpError> {
+  const httpClient = new HttpClient(globalConfig.baseUrl)
+  return httpClient.post(`/api/user-management-system/v1/accounts`, {
+    email,
+    password,
+    password_confirmation: passwordConfirmation,
+    token,
+  })
+}
+
+export async function sendPasswordResetEmail(email: string): Promise<void | HttpError> {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)